### PR TITLE
Adding Contributors Section in Readme

### LINKED ISSUE #11371: Adding Contributors Section in Readme
## Feature Request

I would suggest that there should be a token of appreciation for all the contributors out there to this repository. Therefore, we can add a contributor's section in the documentation i.e. readme below. 

If permits, I would like to take on this issue under Open Source Day'21.

### PR DESCRIPTION
**Pull request by LeeGuoPing**
Fixes #11371 .

Changes proposed in this pull request:
-
-
-

### CODE REVIEW

**Comment by LeeGuoPing** [comment]
@totalo I should only add a link `https://api.github.com/repos/apache/shardingsphere/contributors?page=1&per_page=10000`, or add their ID under the contributors section.  But, If add ID, everytime come a new contributros, we will change the readme.md

**Comment by terrymanu** [comment]
The link in GitHub maybe not enough, how about put the contributor list into page `https://shardingsphere.apache.org/community/cn/team/`?

I prefer use GitHub API to dynamic the contributor list and put the picture in the page, and then put this page link into GitHub READMEmd

**Comment by terrymanu** [comment]
The link in GitHub maybe not enough, how about put the contributor list into page `https://shardingsphere.apache.org/community/cn/team/`?

I prefer use GitHub API to dynamic the contributor list and put the picture in the page, and then put this page link into GitHub READMEmd

**Comment by terrymanu** [comment]
There is a demo in Apache APISIX, FYI: https://apisix.apache.org/team/

**Comment by totalo** [comment]
> 

I think we can make use of the data from this api instead of just attaching this link. Or as @terrymanu  said using APISIX as an example.

**Comment by totalo** [comment]
> @totalo I should only add a link `https://api.github.com/repos/apache/shardingsphere/contributors?page=1&per_page=10000`, or add their ID under the contributors section. But, If add ID, everytime come a new contributros, we will change the readme.md

I think we can make use of the data from this api instead of just attaching this link. Or as @terrymanu said using APISIX as an example.

**Review by terrymanu — CHANGES_REQUESTED** [review]
Just put raw API is not read friendly, we need to consider how to display the contributors.

**Comment by LeeGuoPing** [comment]
> Just put raw API is not read friendly, we need to consider how to display the contributors.

Can I use html, I want to call the API, then display their photos, click their photos foward to their's github

**Comment by LeeGuoPing** [comment]
I use markdown, but now call the raw API,because of the API has rate limit, only 60 times per hour @terrymanu

**Comment by zjcnb** [comment]
Refer to : https://docs.github.com/cn/rest/reference/repos#list-repository-contributors 
`List repository contributors`
I think this way have a problem, becase the github api only response 500 contributors by `https://api.github.com/repos/apache/shardingsphere/contributors?page=1&per_page=10000`.

File: docs/community/content/team/_index.en.md [hunk 1]
```diff
@@ -9,55 +9,377 @@ chapter = true
...
+        <td>
+            <b>Jun Zhao</b>
+        </td>
+        <td>
+            <b>Liang Zhang (Chair)</b>
+        </td>
+    </tr>
+    <tr>
+        <td>
+            <a href="https://github.com/kimmking" target="_blank">
```
Comment (zjcnb) [inline docs/community/content/team/_index.en.md:75]: It 's wrong, Please change to https://github.com/beckhampu and https://avatars.githubusercontent.com/u/14846369?v=4

File: docs/community/content/team/_index.en.md [hunk 2]
```diff
@@ -9,55 +9,377 @@ chapter = true
...
+            <b>Hongtao Gao</b>
+        </td>
+        <td>
+            <b>Juan Pan</b>
+        </td>
+        <td>
+            <b>Jun Zhao</b>
+        </td>
+        <td>
+            <b>Liang Zhang (Chair)</b>
```
Comment (zjcnb) [inline docs/community/content/team/_index.en.md:70]: Please use Chinese` （）`

Comment (LeeGuoPing) [reply docs/community/content/team/_index.en.md:70]: I don't change this place, copy from before

**Comment by LeeGuoPing** [comment]
preview like below:
image
image

**Review by terrymanu — APPROVED** [review]

**Comment by terrymanu** [comment]
Cool

**Pull request merged**